src/GameStart/index.js

```diff
@@ -71,6 +75,35 @@ const GameStart = ({
           />
         </Grid>
       </Grid>
+      <Grid container justify="center" item xs={12} sm={4}>
+        <Grid>
+          <Button variant="contained" color="primary" onClick={onClickStart}>
+            <FormattedMessage
+              id="gameStart.startTeaching3"
+              defaultMessage="Start teaching Parallelograms to {studentName}"
+              values={{ studentName: studentName.replace(/ .*/, "") }}
+            />
+          </Button>
+        </Grid>
+        <Grid>
+          <Button variant="contained" color="primary" onClick={onClickStartMamiferes}>
+            <FormattedMessage
+              id="gameStart.startTeaching1"
+              defaultMessage="Start teaching Mammiferes to {studentName}"
+              values={{ studentName: studentName.replace(/ .*/, "") }}
+            />
+          </Button>
+        </Grid>
+        <Grid>
+          <Button variant="contained" color="primary" onClick={onClickStartAdverbs}>
+            <FormattedMessage
+              id="gameStart.startTeaching2"
+              defaultMessage="Start teaching adverbs to {studentName}"
+              values={{ studentName: studentName.replace(/ .*/, "") }}
+            />
+          </Button>
```

Comment:
I find that there is too much space between the buttons. I imagine it would look better to have the button regrouped, at the level of the bubbles. Come see me if you don't know how to change that style.

![screenshot from 2018-09-19 10-06-33](https://user-images.githubusercontent.com/8818081/45740768-47750380-bbf6-11e8-8e14-ae56e572dfe5.png)